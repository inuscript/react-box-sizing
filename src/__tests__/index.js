import React from 'react'
import assert from 'assert'
import renderer from 'react-test-renderer'
import { BorderBox, ContentBox, PaddingBox } from '../'
import { styleSheet } from 'glamor'

describe('snapshot', () => {
  it('BorderBox', () => {
    const tree = renderer.create(
      <BorderBox><div>This is Test</div></BorderBox>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('ContentBox', () => {
    const tree = renderer.create(
      <ContentBox><div>This is Test</div></ContentBox>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('PaddingBox', () => {
    const tree = renderer.create(
      <ContentBox><div>This is Test</div></ContentBox>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})