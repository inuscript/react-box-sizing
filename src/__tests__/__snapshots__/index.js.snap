exports[`snapshot BorderBox 1`] = `
<div
  data-css-zt4q4s="">
  <div>
    This is Test
  </div>
</div>
`;

exports[`snapshot ContentBox 1`] = `
<div
  data-css-x8cdq="">
  <div>
    This is Test
  </div>
</div>
`;

exports[`snapshot PaddingBox 1`] = `
<div
  data-css-x8cdq="">
  <div>
    This is Test
  </div>
</div>
`;
